index.html

```diff
@@ -1,21 +1,156 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
+  <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
 
-    <title>Responsive grid project</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title> Prickles | Judith Ricketts </title>
+  <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>       <!-- ?? brought in in CSS -->
+  <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">      <!-- ?? brought in in CSS -->
+  <link rel="stylesheet" href="css/style.css">
+  
+  <link rel="stylesheet" href="css/normalize.css">  
+  <!-- Icon for return to teh top-->
+<link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
+
+<!--   javasrio fo the returb to the top  -->
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
 
 <body>
+  <main class="grid">
+      
+      <section class="section sh"> 
+        <img src="img/search.png" alt="search" width="18" height="18"><input type="text" id="search" placeholder="Search.." name="search">
+    
+      </section>
+      <section class="section bk"> <img src="img/basket.png" alt="shopping basket"></section>
+      <section class="section l">PRICKLES & CO.</section>
+      <section class="section nv"> <img src="img/burger.png" alt="burger"  width="30" height="25"></section>
+      <section class="section st"> we deliver fast to your doorstep </section>
+      
+      <section class="section bg1 "> 
+         <li> <h5> ESTB </h5>  <img src="img/logo_.png" alt="logo"  width="180" height="100"> <h5>2023</h5>
+             <h1>PRICKLES & CO</h1> 
+                <p>BRING NATURE INDOORS<p> </li> 
+                  <button type="submit" class="shop submit button " onclick="window.open('https://lovespictures.com/','_blank')"> Shop Now</button>
+      </section>
 
-    <!-- Add your markup here -->
 
+      <section class="section i1a "> 
+        <li><h6> C A C T I </h6>  <img src="img/dash.png" alt="logo"  width="30" height="20"></li>
+        <button type="submit" class="shop collection" onclick="window.open('#')"> Shop Collection</button>
```

Comment:
For buttons, avoid naming their classes after what they say/do. Most buttons are repeated across a website - perhaps you have 1, 2 or 3 button styles maximum that re-occur on different pages. Because of this, we tend to give buttons class names like `button--primary`, `button--secondary`, or sometimes `button--solid` and `button--outline`... you get the idea. This way they can be re-used throughout, without any concern for what the wording is inside the button.

Reply:
(in case you're wondering why I'm using `--` in the class names, I'm following the [BEM notation](https://css-tricks.com/bem-101/))